lib/src/build_importer.dart

```diff
@@ -53,12 +54,26 @@ class BuildImporter extends sass.AsyncImporter {
   /// If neither exists, returns an empty list.
   Future<List<AssetId>> _tryImport(String import) async {
     final imports = <AssetId>[];
-    final partialId = new AssetId.resolve(
-        p.url.join(p.dirname(import), '_${p.basename(import)}'),
-        from: _buildStep.inputId);
+    AssetId partialId;
+    if (_includePath == null) {
```

Comment:
It seems that the only difference between the two cases is where the directory is rooted. Could we give a BuildImporter a default root when it is created without an `_includePath` and then keep these as a single codepath?

Reply:
I'll have to think about it, but I think that there are a lot of different possibilities that the first case is dealing with that don't involve a default root. That would include resolution of package: urls and relative imports. The version with _includePath has to attempt to import using "asset" and a non-relative directory. It makes a lot of assumptions I think that the main case doesn't / shouldn't.

With that said I'm not 100% well versed on asset resolution in build_runner so a secondary opinion would be welcome.